### PR TITLE
add psutil and rfc3339

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: py3-psutil.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - python3
+      - linux-headers
+      - python3-dev
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```
Comment (dlorenc) [inline py3-psutil.yaml:19]: I think we can drop the repos and keyring right?

Comment (dlorenc) [inline py3-psutil.yaml:19]: I think we can drop the repos and keyring right?

Comment (developer-guy) [reply py3-psutil.yaml]: yep, these just added for local testing purposes, thanks for the warning.

Comment (developer-guy) [reply py3-psutil.yaml]: yep, these just added for local testing purposes, thanks for the warning.

**Review by dlorenc — APPROVED** [review]

**Pull request merged**